src/Native/System.IO.Compression.Native/ZLib-1.2.3/ChangeLog

```diff
@@ -0,0 +1,855 @@
+
+                ChangeLog file for zlib
+
+Changes in 1.2.3 (18 July 2005)
```

Comment:
I understand this is a meant to be just a step in the journey, and I see you commented in the PR description about possible next steps.  It's still a bit sad that we're still moving forward with an implementation that's over 10 years old when a much newer solution is available.  It'd be great to use newer source at the very least, and ideally just use some official binary distribution instead.  I like the idea of just referencing the latest in the NuGet feed and using that, rather than building a copy from decade-old source.